libraries/MultisignatureWallet/contracts/MultisigWallet.aes

```diff
@@ -1,331 +1,276 @@
+          map_change_requirement_confirmations = {}}
+
+
+
+    public stateful function init_owner(owner : address) = 
+        only_owner()
+        only_not_configured()
+
+        add_new_owner(owner)
 
     public stateful function configure() = 
-        onlyOwner()
-        require(state.required =< state.ownersCount, "'Required' should be lower than owners count!")
+        only_owner()
+        require(state.required =< state.owners_count, "'Required' should be lower than owners count!")
 
         put(state{configured = true})
 
-    public stateful function voteChangeRequirement(_newRequired: int, _vote: bool) =
-        onlyOwners()
+    public stateful function vote_change_requirement(new_required: int, vote: bool) =
+        only_owners()
         // validate requirements
-        validateRequirement(_newRequired, state.ownersCount)
+        validate_requirement(new_required, state.owners_count)
         // is caller already voted
-        hasVotedForRequirementChange(_newRequired, Call.caller)
+        require(!Map.lookup_default((new_required, Call.caller), state.map_change_requirement_votes, false), "Your vote has been already submitted.")
 
-        put(state{mChangeRequirementVotes[(_newRequired, Call.caller)] = true})
+        put(state{map_change_requirement_votes[(new_required, Call.caller)] = true})
 
-        if(_vote)
-            put(state{mChangeRequirementConfirmations[_newRequired] = getRequirementConfirmations(_newRequired) + 1})
+        if(vote)
+            put(state{map_change_requirement_confirmations[new_required] =  Map.lookup_default(new_required, state.map_change_requirement_confirmations, 0) + 1})
 
-        if(getRequirementConfirmations(_newRequired) >= state.required)
-            put(state{mChangeRequirementConfirmations[_newRequired] = 0})
-            put(state{required = _newRequired})
-            resetRequirementVote(_newRequired, 1, state.ownersCount)
+        if(Map.lookup_default(new_required, state.map_change_requirement_confirmations, 0) >= state.required)
+            put(state{map_change_requirement_confirmations[new_required] = 0})
+            put(state{required = new_required})
+            reset_requirement_vote(new_required, 1, state.owners_count)
 
-    public stateful function voteAddOwner(owner : address) =
-        onlyConfigured()
-        onlyOwners()
-        isNotOwner(owner)
-        hasVotedForOwnerAddition(owner)
+    public stateful function vote_add_owner(owner : address) =
+        only_configured()
+        only_owners()
+        is_not_owner(owner)
+        has_voted_for_owner_addition(owner)
         
-        put(state{mAddOwnerVotes[owner] = getValueAddressInt(owner, state.mAddOwnerVotes) + 1})
-        put(state{mAddOwnerConfirmations[(owner,Call.caller)] = true}) // set this value to true because current owner make a vote 
+        //put(state{map_add_owner_votes[owner] = get_value_address_int(owner, state.map_add_owner_votes) + 1})
```

Comment:
Can we remove this commented code as it is not relevant no more, as far as I can see?